src/main/java/Main.java

```diff
@@ -0,0 +1,43 @@
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import lombok.extern.slf4j.Slf4j;
+import model.ResponseModel;
+import service.OptionCli;
+import service.PrintWeather;
+
+import java.io.IOException;
+
+@Slf4j
+public class Main {
+    public static void main(String[] args) {
+
+        OptionCli optionCli = new OptionCli(args);
+
+        try {
+            if (optionCli.getCmd().hasOption("h") || optionCli.getCmd().getOptions().length < 1) {
+                optionCli.helpBuild();
+            } else {
+                String queryUrl = optionCli.optionBuildQuery();
+                ObjectMapper mapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+                HttpResponse<String> httpResponse = Unirest.get(queryUrl).asString();
+                String response = httpResponse.getBody();
+                ResponseModel responseModel = mapper.readValue(response, ResponseModel.class);
```

Comment:
wykorzystujesz tu biblioteke Unirest. 
pierwszy wynik w google pokazuje że nie używasz polowy dostępnych featurów
```
Unirest.get("http://httpbin.org")
                .queryString("fruit", "apple")
                .queryString("droid", "R2D2")
                .asJson();
```

`asJson()` zakładam że automatycznie zmapuje Ci do `HttpResponse<ResponseModel>` i wystarczy zrobić `getBody()`
`requestParam` pozwala na budowania query które robisz ręcznie `StringBuilderem`

spróbuj to przerobić w tej zaproponowanej przeze mnie klasie